nubis/puppet/files/rc.local

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```

Comment:
Does it need a `service confd restart` here to replicate what you removed in consul-autojoin.sh?